lib/opensusebasetest.pm

```diff
@@ -401,6 +396,8 @@ sub wait_boot {
             # check_screen timeout
             die "needle 'grub2' not found";
         }
+        # confirm default choice
+        send_key 'ret';
```

Comment:
no problem, trust me :)

Reply:
Actually not just "no problem" but should fix something. wait_boot encryption unlocking on ppc64le !storage-ng was broken since the introduction of disable_grub_timeout, see https://openqa.suse.de/tests/1408031#step/reboot_gnome/16 . That should be fixed. Let's see if https://openqa.suse.de/tests/1462618 will be able to show this already (still running).

Reply:
… and it's fixed 🎆